src/node_config.h

```diff
@@ -0,0 +1,17 @@
+#ifndef SRC_NODE_CONFIG_H_
+#define SRC_NODE_CONFIG_H_
+
+#include "node.h"
+
+namespace node {
+
+#if defined(NODE_HAVE_I18N_SUPPORT)
+  extern bool flag_icu_data_dir;
+#endif
+
+namespace config {
+
+}  // namespace config
```

Comment:
What's the point of the `config` namespace when it's empty?  This header file seems a little superfluous to begin with; I'd stick the bool in e.g. node_internals.h.